config/boot.rb

```diff
@@ -2,34 +2,42 @@
 $LOAD_PATH.unshift File.expand_path(File.join(File.dirname(__FILE__), '..', 'robots'))
 
 require 'logger'
-
 # Load the environment file based on Environment.  Default to development
 environment = ENV['ROBOT_ENVIRONMENT'] ||= 'development'
 ROBOT_ROOT = File.expand_path(File.dirname(__FILE__) + '/..')
 ROBOT_LOG = Logger.new(File.join(ROBOT_ROOT, "log/#{environment}.log"))
 ROBOT_LOG.level = Logger::SEV_LABEL.index(ENV['ROBOT_LOG_LEVEL']) || Logger::INFO
 
-# Load core robot services
-require 'dor-services' # FIXME: do we really need dor-services?   Just for Dor::Config ??
 require 'lyber_core'
 LyberCore::Log.set_level(ROBOT_LOG.level)
-require 'robots'
 
-# Load local environment configuration
-env_file = File.expand_path(File.dirname(__FILE__) + "/environments/#{environment}")
-puts "Loading config from #{env_file}"
-require env_file
+require 'config'
+Config.setup do |config|
+  config.use_env = true
+  config.env_prefix = 'SETTINGS'
+  config.env_separator = '__'
+end
+Config.load_and_set_settings(Config.setting_files(File.dirname(__FILE__), environment))
```

Comment:
i'm not sure i understand what's happening here (i didn't see equivalent code in the code being removed from preservation_robots or in dor-services).

Reply:
This is from having added the `config` gem.  Without rails initialization, we have to load the config oursevles.  Before this line `Settings` is not defined.  Afterwards, it is populated.